src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/FileVersionInfoTest.cs

```diff
@@ -102,6 +102,21 @@ public void FileVersionInfo_NonExistentFile_FileNotFound()
                 FileVersionInfo.GetVersionInfo(Path.Combine(Directory.GetCurrentDirectory(), TestNotFoundFileName)));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapNotUapAot, "Don't want to create temp file in app container current directory")]
```

Comment:
Why is this not just `TargetFrameworkMonikers.Uap`? 

I would think we do not want to create temp file in app container for both UapAot and UapNotUapAot - both of them are running in app container.

Reply:
I don't know much about UAP, this was on @danmosemsft's suggestion

Reply:
Ok. Could you please change this to just Uap?